tests/test_sandbox/test_sandbox_backends.py

```
@@ -211,18 +211,38 @@ async def test_exec_runs_command(self, mock_shell):
         assert result.return_code == 0
         await sb.stop()
 
+    @patch("nemo_evaluator.sandbox.local.os.killpg")
+    @patch("nemo_evaluator.sandbox.local.os.getpgid", return_value=4321)
     @patch("nemo_evaluator.sandbox.local.asyncio.create_subprocess_shell")
-    async def test_exec_timeout(self, mock_shell):
+    async def test_exec_timeout(self, mock_shell, mock_getpgid, mock_killpg):
```

Ruff (0.15.18)

[warning] Unused method argument: `mock_getpgid`

(ARG002)